### PR TITLE
[PXCT-80] Nano Matter - Videos/Support Tutorials

### PR DESCRIPTION
**Pull request by mcmchris**
## What This PR Changes
- Convert vertical videos created for Marketing into tutorials for product Launch

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/add-device-2.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/add-device-2.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/add-device.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/add-device.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/bsp-install-2.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/bsp-install-2.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/diagram-v4.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/diagram-v4.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/download.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/download.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/matter-fan.gif
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/matter-fan.gif:1]: nice to have. increase frames

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/matter-setup-2.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/matter-setup-2.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/serial-monitor-v4.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/serial-monitor-v4.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/thumbnail-v4.png
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/assets/thumbnail-v4.png:1]: checked

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+---
+title: 'Matter Smart Fan'
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:2]:
```suggestion
title: 'Matter Smart Fan with the Arduino Nano Matter'
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+difficulty: beginner
+compatible-products: [nano-matter]
+description: 'Learn how to build a Matter fan with speed control.'
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:5]:
```suggestion
description: 'Learn how to build a Matter fan with speed control using the Arduino Nano Matter'
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+## Overview
+
+This tutorial will teach you how to create a Matter fan for keeping you cool.
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:24]:
```suggestion
This tutorial will teach you how to use the Arduino Nano Matter to create a Matter fan to keep you cool. 
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+  - iot-cloud
+---
+
+## Overview
+
+This tutorial will teach you how to create a Matter fan for keeping you cool. 
+
+![Matter fan overview](assets/thumbnail-v4.png)
+
+Thanks to the seamless compatibility of the Nano Matter with almost any Matter network we can easily integrate our fan with Amazon Alexa, Google Assistant, Apple Home, Home Assistant and even custom assistants.
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:28]:
```suggestion
Thanks to the seamless compatibility of the Nano Matter with almost any Matter network we can easily integrate our fan with Amazon Alexa, Google Assistant, Apple Home, Home Assistant and even any professional custom solution.
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+We have prepared a short demo in video format in case you are a visual learner.
+
+<iframe width="100%" height="480" src="https://www.youtube.com/embed/AiFm4T9EfnA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Hardware and Software Requirements
+### Hardware Requirements
+
+- [Nano Matter](https://store.arduino.cc/products/nano-matter) (x1)
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:37]:
```suggestion
- [Arduino Nano Matter](https://store.arduino.cc/products/nano-matter) (x1)
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+![Installing the Silicon Labs core in the Arduino IDE](assets/bsp-install-2.png)
+
+## Project Setup
+
+### Schematic Diagram
+
+Use the following connection diagram for the project:
+
+![Project wiring diagram](assets/diagram-v4.png)
+
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:73]: add two lines explanation about the general connection or **the main thing the user must take attention**

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    Serial.println();
+    digitalWrite(LED_BUILTIN, LED_BUILTIN_INACTIVE);
+    // This function will not return
+    // The device will restart once decommissioning has finished
+    Matter.decommission();
+  }
+}
+```
+
+This is a very simple code that consist of the Arduino basic sketch parts:
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:192]: improve the phrase, a little bit broken

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+- In the `setup()` function we initialize the Matter connectivity and the fan output pin. 
+- In the `loop()` function we listen to any fan control command sent from the smartphone or voice assistant and then adjust the fan speed setting the PWM signal accordingly.
+
+Once you uploaded the example code to the Nano Matter, open the Serial Monitor and reset the board.
+
+![QR Code URL](assets/serial-monitor-v4.png)
+
+There you will find the URL that generates the QR for the Matter device commissioning.
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:200]: there is not correct

```suggestion
After the reset you will find on the serial port the URL that generates the QR for the Matter device commissioning.
```

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+
+### Commissioning
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:202]: any way to improve this header? Device commissioning? we need a way to explain the less advanced users what is commissioning

File: content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+
+## Final Results
+
+Finally, you will be able to control the fan from your smartphone, hub or asking your personal assistant.
+
+![Matter fan final result](assets/animation.gif)
+
+## Conclusion
+
+In this tutorial we have learned how to create a Matter enabled fan that can be controlled from our smartphone and personal assistant. The Nano Matter allows us to seamlessly integrate the fan as a commercial product with our current smart home ecosystem.
```
Comment (jcarolinares) [inline content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md:220]:
```suggestion
In this tutorial we have learned how to create a Matter enabled fan that can be controlled from our smartphone and personal assistant. The Nano Matter allows us to seamlessly integrate our own custom or old fan as a commercial product with our current smart home ecosystem.
```

**Review by jcarolinares — CHANGES_REQUESTED** [review]
@mcmchris 

Please take a look at the review done for **the matter fan** and replicate that feedback if possible in the rest of tutorials. Special attention please to the tittle and description format suggested, it is important for SEO reasons :+1:

**Comment by mcmchris** [comment]
All tutorials are updated with your feedback. Thanks

**Review by jcarolinares — APPROVED** [review]
Ready :)

**Comment by mcmchris** [comment]
Merging!

**Pull request merged**